frontend/src/components/course/course.tsx

```diff
@@ -0,0 +1,51 @@
+import defaultCourseImage from 'assets/img/default-course-image.jpeg';
+import { DataStatus } from 'common/enums/enums';
+import { FC } from 'common/types/types';
+import { Image, Spinner } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  useParams,
+} from 'hooks/hooks';
+import { courseActions } from 'store/actions';
+
+import styles from './styles.module.scss';
+
+const Course: FC = () => {
+  const { course, dataStatus } = useAppSelector((state) => state.course);
+  const { id } = useParams();
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(courseActions.getCourse({ id: Number(id) }));
+  }, [id]);
+
+  if (dataStatus === DataStatus.PENDING) {
+    return <Spinner />;
+  }
+
+  return (
```

Comment:
Won't it be better to show some placeholder in case if we haven't found course by `id`?

Reply:
agreed